SConstruct

```diff
@@ -1208,6 +1208,7 @@ else:
 			"/permissive-", # disable permissive mode, which also enables standard compliant two phase name lookup
 			# required in when permissive mode is off, for defining constants like M_PI used by OpenVDB
 			"/D_USE_MATH_DEFINES",
+			"/Zc:externC-",  # suppress errors from Boost::interprocess Windows SDK mismatches
```

Comment:
I don't think we actually need to use `interprocess` any more, so I've opened a PR to remove it : https://github.com/ImageEngine/cortex/pull/1106. I suggest we merge that and drop this commit.